mysql-test/suite/versioning/t/replace.test

```diff
@@ -14,7 +14,38 @@ eval create or replace table t(
 
 insert t values (1, 2);
 replace t values (1, 3);
-select *, current_row(row_end) as current from t for system_time all order by x;
+replace t values (1, 3);
+select *, current_row(row_end) as current, count(row_start) = 1 as unique_start,
```

Comment:
How can you check `row_end` in group by `row_start`?

Reply:
Well, it works, at least, until row_start is unique. Maybe it is really better to make separate query for unique checking